doc/advancedFeatures.rst

```diff
@@ -39,8 +48,24 @@ Because the hot start process will store all the previous "restarted" iterations
 
 
 
-.. Time limit
-.. ----------
+Time limit
```

Comment:
This only works for SNOPT because it's the only one that has the interrupt feature (via user requested optimization termination). Although it's possible that other optimizers may also allow this, they are not implemented.

Reply:
Right, just edited it